Dockerfile

```diff
@@ -1,44 +1,31 @@
 ARG FROM
 FROM ${FROM} as builder
 
-RUN apk add --no-cache \
-      bash \
-      build-base \
-      cargo \
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update -qq \
+  && apt-get install \
+      --yes -qq --no-install-recommends \
+      build-essential \
       ca-certificates \
-      cmake \
-      cyrus-sasl-dev \
-      git \
       graphviz \
-      jpeg-dev \
       libevent-dev \
       libffi-dev \
-      libxslt-dev \
-      make \
-      musl-dev \
-      openldap-dev \
-      postgresql-dev \
-      py3-pip \
+      libjpeg-dev \
+      libldap-dev \
+      libsasl2-dev \
+      libxslt1-dev \
+      libxml2-dev \
+      postgresql-13 \
       python3-dev \
+      python3-pip \
+      python3-venv \
```

Comment:
```suggestion
      python3-venv \
      libpq-dev \
      libssl-dev \
```

Reply:
I'll test as well shortly.  @cimnine can you please make the changes to test via CI?

Reply:
Did you remove the `psycopg2-binary==2.9.3` line from the Netbox `requirements.txt` or is `psycopg2==2.9.3` simply added as an additional dependency?

Reply:
I didn't need to remove it from the netbox requirements.